Document/0x06a-Platform-Overview.md

```diff
@@ -131,27 +131,28 @@ A language.lproj folder is defined for each language that the application suppor
 
 On a jailbroken device, you can recover the IPA for an installed iOS app using [IPA Installer](https://github.com/autopear/ipainstaller "IPA Installer"). Note that during mobile security assessments, developers will often provide you with the IPA directly. They could send you the actual file, or provide access to the development specific distribution platform they use e.g. [HockeyApp](https://hockeyapp.net/ "HockeyApp") or [Testflight](https://developer.apple.com/testflight/ "Testflight").
 
+
 #### App Structure on the iOS File System
 
-Since iOS 8, changes were made to the way an application is stored on the device. On versions before iOS 8, applications would be unpacked to a folder in the /var/mobile/applications/ folder. The application would be identified by its UUID (Universal Unique Identifier), a 128-bit number. This would be the name of the folder in which we will find the application itself. Since iOS 8 this has changed however, so we will see that the static bundle and the application data folders are now stored in different locations on the filesystem. These folders contain information that we will need to closely examine during application security assessments.
+iOS 8 introduced changes to the way an application is stored on a device. In the earlier versions of iOS applications would be unpacked to a folder in the /var/mobile/applications/ folder. An application would then be identified by its UUID (Universal Unique Identifier), a 128-bit number that will be the name of the folder where the application can be found. Since iOS 8 release static bundle and application data folders are now stored in different locations in the file system. These folders contain information that we must be closely examined during application security assessments.

```

Comment:
Should remove the second "folder", maybe -> In the earlier versions of iOS, applications would be unpacked to the folder /var/mobile/applications/.

I would remove release or write "the iOS 8 release" and add "the" -> Since iOS 8 the static bundle and application data folders are now stored in different locations in the file system. 

Remove "we" -> These folders contain information that must be closely examined during application security assessments.